bin/pasta_prepare_evaluation.py

```diff
@@ -13,11 +13,23 @@
 the COPYING file in the top-level directory.
 """
 
+from analyses import response_analysis
+from pypasta.LinuxMaintainers import LinuxMaintainers
+from pypasta import LinuxMailCharacteristics, load_linux_mail_characteristics, \
+    email_get_header_normalised, email_get_from
+
 import argparse
+from ast import literal_eval
 import csv
+import dask.dataframe as dd
+import email
+import flat_table
+import numpy as np
 import os
+import pandas as pd
```

Comment:
Same here, please update the docker files and test if the image is still able to run pasta.

Reply:
Sure